sdk/storage/storage-blob-changefeed/test/blobchangefeedclient.spec.ts

```diff
@@ -203,3 +206,403 @@ describe("BlobChangeFeedClient: Change Feed not configured", async () => {
     assert.ok(exceptionCaught);
   });
 });
+
+describe("Change feed event schema test", async () => {
+  let recorder: Recorder;
+
+  before(async function (this: Context) {
+    if (process.env.CHANGE_FEED_ENABLED === "1" && !isPlaybackMode()) {
+      this.skip();
+    }
+  });
+
+  beforeEach(async function (this: Context) {
+    recorder = record(this, recorderEnvSetup);
+  });
+
+  afterEach(async function () {
+    await recorder.stop();
+  });
+
+  it("Event schema v1 test", async () => {
+    const eventSchemaV1 = path.join("test", "resources", "EventSchemaV1.json");
+    const eventData = await streamToString(fs.createReadStream(eventSchemaV1));
+    const eventObject = JSON.parse(eventData);
+    const changeFeedEvent = rawEventToBlobChangeFeedEvent(eventObject);
+
+    // Assert
+    assert.equal(1, changeFeedEvent.schemaVersion);
+    assert.equal(
+      "/subscriptions/dd40261b-437d-43d0-86cf-ef222b78fd15/resourceGroups/haambaga/providers/Microsoft.Storage/storageAccounts/HAAMBAGA-DEV",
+      changeFeedEvent.topic
+    );
+    assert.equal(
+      "/blobServices/default/containers/apitestcontainerver/blobs/20220217_125928329_Blob_oaG6iu7ImEB1cX8M",
+      changeFeedEvent.subject
+    );
+    assert.equal("BlobCreated", changeFeedEvent.eventType);
+    assert.equal(
+      new Date("2022-02-17T12:59:41.4003102Z").valueOf(),
+      changeFeedEvent.eventTime.valueOf()
+    );
+    assert.equal("322343e3-8020-0000-00fe-233467066726", changeFeedEvent.id);
+    assert.equal("PutBlob", changeFeedEvent.data.api);
+    assert.equal("f0270546-168e-4398-8fa8-107a1ac214d2", changeFeedEvent.data.clientRequestId);
+    assert.equal("322343e3-8020-0000-00fe-233467000000", changeFeedEvent.data.requestId);
+    assert.equal("0x8D9F2155CBF7928", changeFeedEvent.data.etag);
+    assert.equal("application/octet-stream", changeFeedEvent.data.contentType);
+    assert.equal(128, changeFeedEvent.data.contentLength);
+    assert.equal("BlockBlob", changeFeedEvent.data.blobType);
+    assert.equal("https://www.myurl.com", changeFeedEvent.data.url);
+    assert.equal(
+      "00000000000000010000000000000002000000000000001d",
+      changeFeedEvent.data.sequencer
+    );
+  });
+
+  it("Event schema v3 test", async () => {
```

Comment:
Do you miss schema v2 test?

Reply:
This test cases all follow .Net SDK's cases. No v2 test in .Net SDK either..